.github/workflows/ci.yml

```diff
@@ -26,6 +26,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
```

Comment:
not even sure this is necessary for runner- these tests succeeded without it for me